presto-hive/src/test/java/com/facebook/presto/hive/metastore/glue/TestHiveClientGlueMetastore.java

```diff
@@ -139,4 +159,57 @@ public void testGetPartitions()
             dropTable(tablePartitionFormat);
         }
     }
+
+    @Test
+    public void testTableWithoutStorageDescriptor()
+    {
+        // StorageDescriptor is an Optional field for Glue tables. Iceberg and Delta Lake tables may not have it set.
+        SchemaTableName table = temporaryTable("test_missing_storage_descriptor");
+        DeleteTableRequest deleteTableRequest = new DeleteTableRequest()
+                .withDatabaseName(table.getSchemaName())
+                .withName(table.getTableName());
+        AWSGlueAsync glueClient = AWSGlueAsyncClientBuilder.defaultClient();
+        try (Transaction transaction = newTransaction()) {
```

Comment:
Is this `Transaction transaction = newTransaction()` necessary? Looks like it's not used.

Reply:
You're right. It's not used.